lib/devise-security/models/old_password.rb

```diff
@@ -1,6 +1,28 @@
 # frozen_string_literal: true
 
-require 'active_record'
-class OldPassword < ActiveRecord::Base
-  belongs_to :password_archivable, polymorphic: true
+if DEVISE_ORM == :active_record
```

Comment:
IMO instead of doing this we should have two files, one for the AR version and one for the Mongoid version. We then `require` the appropriate one based on this value

Reply:
I've splitted into two different files:

`lib/devise-security/models/active_record/old_password.rb`
`lib/devise-security/models/mongoid/old_password.rb`

and load the right one inside `devise-security.rb` with
`require "devise-security/models/#{DEVISE_ORM}/old_password" if defined? DEVISE_ORM`